Package.swift

```diff
@@ -42,18 +42,27 @@ let package = Package(
             dependencies: [
                 .product(name: "Vapor", package: "vapor"),
                 .product(name: "JWTKit", package: "jwt-kit"),
```

Comment:
I'd update to JWTKit 5, up to you whether here or in another PR

Reply:
Absolutely, it's my next priority, but I'll do it in another PR